src/i18n/pages/pickupDetails.ts

```diff
@@ -0,0 +1,19 @@
+export const pickupDetailsEs = {
+    title: "Detalle recogida",
```

Comment:
I think this should be `Detalles de la recogida`

Reply:
This one I left as it was for now, because it gets a little too big for the page title